app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -25,7 +25,7 @@ class Api::V2::ContentViewVersionsController < Api::V2::ApiController
     def index
```

Comment:
I just looked at the existing controller tests ... there's a good opportunity for some copy/paste programming to test sorting by version - mind adding something basic?